app/views/tag_wranglings/index.html.erb

```diff
@@ -104,32 +104,34 @@
 
             <tbody>
               <% @tags.each do |tag| %>
-                <tr>
-                  <th scope="row" title="<%= ts('tag') %>">
-                    <%= check_box_tag 'selected_tags[]', tag.id, nil, id: "selected_tags_#{tag.id}" %>
-                    <%= label_tag "selected_tags_#{tag.id}", "#{tag.name}" %>
-                  </th>
-
-                  <td title="<%= ts('created') %>"><%= l(tag.created_at.to_date) %></td>
-
-                  <td title="<%= ts('canonical?') %>">
-                    <% if tag.canonical? %>
-                      <%= ts('Yes') %>
-                    <% else %>
-                      <%= check_box_tag 'canonicals[]', tag.id, tag.canonical?, id: "canonicals_#{tag.id}" %>
-                    <% end %>
-                  </td>
-
-                  <td title="<%= ts('taggings') %>"><%= tag.taggings_count %></td>
-
-                  <td>
-                    <ul class="actions" role="navigation">
-                      <li><%= link_to ts('Edit'), {controller: :tags, action: :edit, id: tag} %></li>
-                      <li><%= link_to ts('Wrangle'), {controller: :tags, action: :wrangle, id: tag} %></li>
-                      <li><%= link_to ts('Works'), {controller: :works, action: :index, tag_id: tag} %></li>
-                    </ul>
-                  </td>
-                </tr>
+                <% if tag.unwrangled? %>
```

Comment:
A non-solution to an actual problem.

Since the tags are now loaded from ES, they aren't always up to date. Typically, after a wrangler does any operation, they're returned to the page, nothing have been reindexed yet, and so the tag they've just wrangled is still there as if they did nothing.

I'm wondering if, while computing the numbers in the sidebar in ES is definitely good (they don't need to be perfectly accurate and all, especially as they're already cached), having the main table, with operations happening all the time, not getting the freshest info from the DB is not a cure worse than the disease, gaining a bit of performance over the price of so many potential visual bugs and bug reports.

Reply:
Based on the story's description, I think we'd like to try at least on staging. However, I could see the scenario you describe being quite difficult... @sarken thoughts?

Reply:
So fwiw, this basically happens already -- e.g., I just wrangled two tags to my fandom, and they were still in my fandom's unwrangled bin (`/tags/Fandom%20Name/wrangle?show=freeforms&status=unwrangled`) when the page reloaded. Wranglers are pretty used to changes not being immediate, so pulling the contents of the bins from Elasticsearch should be fine.